python/tvm/driver/tvmc/compiler.py

```diff
@@ -0,0 +1,407 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Provides support to compile networks both AOT and JIT.
+"""
+import argparse
+import logging
+import tarfile
+from pathlib import Path
+
+import tvm
+from tvm import autotvm
+from tvm import relay
+from tvm._ffi.runtime_ctypes import TVMContext
+from tvm.contrib import cc
+from tvm.contrib import util
+from tvm.relay.op.contrib import get_pattern_table
+
+from . import common, frontends
+from .main import register_parser
+
+# A dictionary of target aliases to simplify the command lines provided by end users
+TARGET_ALIASES = {
+    "aarch64": "llvm -device=arm_cpu -mtriple=aarch64-linux-gnu -mattr=+neon"
+}
```

Comment:
We should reuse TVM target (`python/tvm/target/target.py`) instead of maintaining a duplicate list here. 

cc @junrushao1994 for comments as this is also related to the target tag.

Reply:
It is really a consequence of the original development being on a separate repo, and making sure we have access to the targets we wanted for our testing purposes.

Now, it makes much more sense to go the way you suggest, I'll change that.

Reply:
@junrushao1994 is there a way to obtain a list of all the registered targets so that we can offer that list in the `python -m tvm.driver.tvmc compile --help`, rather than the user needing to guess what is the expected target tag for the actual hardware they got?

Reply:
@junrushao1994 could you comment? It's fine if we don't have it yet but plan to have in the future. In that case we can leave a TODO here to move forward.

Reply:
TODO added